server.go

```diff
@@ -156,7 +156,13 @@ func main() {
 	})
 
 	r.Get("/folders", func(w http.ResponseWriter, r *http.Request) {
-		data, err := drive.List("/", &wfs.ListConfig{
+		r.ParseForm()
```

Comment:
probably it must be changed to

```
id := r.URL.Query().Get("id")
```

this is GET request, we do not have a form here

Reply:
fixed